src/main/java/org/opentripplanner/inspector/vector/LayerParameters.java

```diff
@@ -10,7 +12,7 @@ public interface LayerParameters<T extends Enum<T>> {
   int MAX_ZOOM = 20;
   int CACHE_MAX_SECONDS = -1;
   double EXPANSION_FACTOR = 0.25d;
-
+  List<String> HIDE_NETWORKS = new ArrayList<>();
```

Comment:
The variable name sounds more like a method name. Maybe HIDDEN_NETWORKS or NETWORKS_TO_HIDE instead. Also, maybe filter out instead of hide as we are not really displaying anything. This could be a Set.

Reply:
Removed hiding networks.